lib/cn/csched_sp3_work.c

```diff
@@ -338,10 +338,10 @@ sp3_work_leaf_garbage(
         /* Node is not big enough to spill.  If there's more than
          * one kvset, then kv-compact.  Otherwise do nothing.
          */
-        if (kvsets > 1)
-            *action = CN_ACTION_COMPACT_KV;
-        else
+        if (kvsets < 4)
             return 0;
+
+        *action = CN_ACTION_COMPACT_KV;
```

Comment:
This change broke droptomb2.  I found with kvt it was triggering kv-compact in each level-1 node for each root spill in all of my 32 trees.  Basically, tiny kvset that's mostly garbage will always trigger this kv-compact, so I increased it to minimum of 4 kvsets to trigger the kv-compact (versus two).